### PR TITLE
Add support for PHP 8.4

### PR DESCRIPTION
**Pull request by ash-jc-allen**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
         "illuminate/database": "^10.0|^11.0",
         "hashids/hashids": "^4.0|^5.0",
         "whichbrowser/parser": "^2.1",
-        "ashallendesign/laravel-config-validator": "^2.6.1"
+        "ashallendesign/laravel-config-validator": "^2.6.1",
+        "laravel/framework": "^10.0"
```
Comment (kyryl-bogach) [inline composer.json:28]: Do we really need this depedency? Wouldn't this mean that it would conflict with Laravel 11 installs?

Comment (kyryl-bogach) [reply composer.json:28]: I basically can't upgrade to `^8.2` because of this.

Comment (ash-jc-allen) [reply composer.json:28]: Hey @kyryl-bogach! Huge thank you for pointing this out to me. I was doing some tinkering the other day and accidentally committed this (I can't believe I did that!) 🤦‍♂️

I've just removed it now, so you should be able to upgrade to v8.2.1 🤞

Sorry about that!

Comment (kyryl-bogach) [reply composer.json:28]: Thank you so much sir! 🙏🏻

**Pull request merged**